src/main/java/io/codelex/exceptions/Exercise2.java

```diff
@@ -0,0 +1,36 @@
+package io.codelex.exceptions;
+
+public class Exercise2 {
+    public void methodA() {
+        //System.out.println(10 / 0);
+        try {
+            methodB();
+        } catch (ArithmeticException e) {
```

Comment:
`After each try{} block put a catch{} block which catches the exception, prints a stack trace, and throws the exception object to its caller.`